### ISSUE TITLE
FR: exit without saving config

### ISSUE THREAD

**Issue by PsyhoBelka** [Rem0o/FanControl.Releases#2889]
Is there possibility to tell the app not saving the config on app close? I'm using it for manually setup the fan speed depending on the system loads and when I turn off pc it every time asked for saving config, which prevents pc to shutdown. Thanks

**Comment by PsyhoBelka** [comment]
implement just one checkbox in settings - don't ask for save on exit. it will be enough, imho.

**Comment by Rem0o** [comment]
Fair enough. I would make it as an opt out thing to keep the existing behaviour by default.
